src/stac_api_benchmark/query.py

```diff
@@ -258,7 +282,7 @@ async def search_with_fc(
     datetime: Optional[str] = None,
     sortby: Optional[List[Dict[str, str]]] = None,
     exclude_ids: Optional[List[str]] = None,
-) -> Tuple[List[Union[Tuple[int, float], Exception]], float]:
+) -> Tuple[List[Result[RunSuccess, RunFailure]], float]:
```

Comment:
Bikeshed comment so take or leave it, `type RunResult = Result[RunSuccess, RunFailure]` (or however you do type aliases in python)

Reply:
lol, i would have done that if i could remember the type alias syntax :)